deployment/united-manufacturing-hub/templates/kafkatopostgresql/deployment.yaml

```diff
@@ -55,11 +55,13 @@ spec:
           image: {{.Values.kafkatopostgresql.initContainer.repository}}:{{.Values.kafkatopostgresql.initContainer.tag}}
           {{- else}}
           image: {{.Values.kafkatopostgresql.initContainer.repository}}:{{.Chart.AppVersion}}
+          {{ if and (hasKey .Values._000_commonConfig.infrastructure.kafka "useSSL") .Values._000_commonConfig.infrastructure.kafka.useSSL }}
```

Comment:
let's test whether all cases work here as expected. I remember we tried something like this before, but were never able to enable or disable it.

Reply:
After solving this https://github.com/united-manufacturing-hub/united-manufacturing-hub/issues/1549 it will be easier to adjust this